src/components/Skeleton/SkeletonConstants.ts

```diff
@@ -1,32 +1,35 @@
 // Fixed names
-export const SKELETON_CIRCLE_TYPE = "circle";
-export const SKELETON_TEXT_TYPE = "text";
-export const SKELETON_RECTANGLE_TYPE = "rectangle";
 export const SKELETON_CUSTOM_SIZE = "custom";
```

Comment:
and this const is still here. Maybe just leave it as it is?

Reply:
removed it also :)